modules/installation/pages/uyuni-proxy-registration.adoc

```diff
@@ -0,0 +1,89 @@
+[[proxy-register]]
+= Proxy Registration
+
+Proxy systems are registered as Salt clients using a bootstrap script.
+
+This procedure describes software channel setup and registering the installed proxy with an activation key as a {productname} client.
+
+[IMPORTANT]
+====
+Before you can select the correct child channels while creating the activation key, ensure you have properly synchronized the openSUSE Leap channel with all the needed child channels and the {productname} Proxy channel.
+====
+
+// ==>>>> uyuni-proxy-stable-leap-151 channel
+
+
+[[proxy-register-procedure]]
+.Procedure: Registering the Proxy
+
+. First create openSUSE Leap and the {productname} Proxy channels with the [command]``spacewalk-common-channels`` command.
+[command]``spacewalk-common-channels`` is part of the [package]``spacewalk-utils`` package:
++
+----
+spacewalk-common-channels \
+opensuse_leap15_1 \
+opensuse_leap15_1-non-oss \
+opensuse_leap15_1-non-oss-updates \
+opensuse_leap15_1-updates \
+opensuse_leap15_1-uyuni-client \
+uyuni-proxy-stable-leap-151
+----
++
+For more information, see xref:client-configuration:clients-opensuse.adoc[].
+
+. Create the openSUSE Leap 15 Uyuni Client Tools Repository for bootstrapping, see xref:client-configuration:creating-a-tools-repository.adoc[].
+. Create an activation key with openSUSE Leap as a base channel and the other channels as child channels.
+For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
++
+
+// // .Proxy Activation Key
+// // image::proxy-key.png[]
+// +
+// ////
+// . From the [guimenu]``Child Channels`` listing select the recommended
+// channels by clicking the ``include recommended`` icon:
+// +
+// * SLE-Module-Basesystem15-SP1-Pool
+// * SLE-Module-Basesystem15-SP1-Updates
+// * SLE-Module-Server-Applications15-SP1-Pool
+// * SLE-Module-Server-Applications15-SP1-Updates
+// * SLE-Module-SUSE-Manager-Proxy-4.0-Pool
+// * SLE-Module-SUSE-Manager-Proxy-4.0-Updates
+// +
+// The [systemitem]``SLE-Product-SUSE-Manager-Proxy-4.0-Updates`` channel is mandatory.
+// +
+// .Base and Child Proxy Channel
+// image::proxy-child.png[]
+
+
+. Modify a bootstrap script for the proxy.
+* Add the GPG key to the [systemitem]``ORG_GPG_KEY=`` parameter.
+For more information, see xref:client-configuration:clients-opensuse.adoc[].
+* OPTIONAL: If you want to run the proxy on a traditional client (system type ``Management``) uncheck [guimenu]``Bootstrap using Salt``.
+Using Salt is the default.
++
+// FIXME: I do not know whether changing to traditional is actually possible
```

Comment:
FMPOV on Uyuni we should support only proxies based on salt clients. @paususe @mcalmer?

If so, this could be removed.